app/src/main/java/com/esaudev/wizeline/data/remote/responses/AvailableBooksResponse.kt

```diff
@@ -0,0 +1,42 @@
+package com.esaudev.wizeline.data.remote.responses
+
+import com.esaudev.wizeline.model.AvailableBook
+
+data class AvailableBooksResponse(
+    val payload: List<AvailableBooksPayload>,
+    val success: Boolean
+)
+
+data class AvailableBooksPayload(
+    val book: String,
+    val maximum_amount: String,
+    val maximum_price: String,
```

Comment:
It would be better to use camelCase in your vars, you can use SerializedName to take a different name from the server response to you local class :D 

```suggestion
@SerializedName("maximum_price")
    val maximumPrice: String,
```